tests/test_rule.py

```diff
@@ -81,8 +81,8 @@ def test_parse_fileobj(self):
         self.assertEqual(2, len(rules))
 
     def test_parse_file(self):
```

Comment:
We should also have a test for the case you just covered. Could you try and add that?

Reply:
How can I run these tests on my laptop?